edx/cc2olx
# [BD-27] [TNL-7948] Fix Static File References

We noticed that not all static links were escaped as part of the conversion process. This PR adds code to recursively process all input going into OLX nodes to automatically escape any static file references.

**JIRA tickets**: TNL-7948

**Merge deadline**: None

**Testing instructions**:

1. Download and try to import a course with static links in various places, including in multiple-choice problems answers (Where we found the problem), discussion topics & other locations
2. Convert the course with cc2olx and verify that all static links were escaped successfully

**Reviewers**
